doc/source/reference/arrays.dtypes.rst

```diff
@@ -188,10 +188,10 @@ Built-in Python types
     (all others)      :class:`object_`
     ================  ===============
 
-    Note that ``str`` refers to either null terminated bytes or unicode strings
-    depending on the Python version. In code targeting both Python 2 and 3
-    ``np.unicode_`` should be used as a dtype for strings.
-    See :ref:`Note on string types<string-dtype-note>`.
+    Note that ``str`` corresponds to UCS4 encoded unicode strings, while
+    ``string`` is an alias to ``bytes_``. The name `np.unicode_` is also
+    available as an alias to `np.str_`, see :ref:`Note on string
```

Comment:
```suggestion
    ``string`` is an alias to ``bytes_``. The name ``np.unicode_`` is also
    available as an alias to ``np.str_``, see :ref:`Note on string
```

Just to get the [rendered html formatted a little more nicely](https://output.circle-artifacts.com/output/job/1f3abf6e-287d-4993-9b91-61fc3b19e3d4/artifacts/0/doc/build/html/reference/arrays.dtypes.html)

Reply:
Ah nice, I was wondering if there's a way to see the rendered docs from the CI build.

Reply:
We usually have an artifact redirecting job that makes it even simpler, but that has been broken for a bit due to a security-hardening attempt for actions that hasn't been completely sussed out yet.